x-pack/solutions/observability/plugins/observability/public/pages/util/synthetics_alert_params_to_kql.ts

```diff
@@ -0,0 +1,120 @@
+/*
```

Comment:
I'd create a folder for the `use_discover_url` hook and place these files somewhere inside that folder so that it's clear that, at least right now, that's the only place where they are used. wdyt?

Reply:
I've created a folder in [this PR](https://github.com/elastic/kibana/pull/233855/files) to have more order.

Reply:
Ok, I will mimic the folder creation in this PR and hopefully it will result in not-too-many merge conflicts for the two patches 😄

Reply:
ba9b777d35bd5627e288ef8b317049b0d8ecd6e8